src/factory/Layer.js

```diff
@@ -231,7 +231,6 @@ export const LayerFactory = {
           service: 'WFS',
           version: lConf.version,
           request: 'GetFeature',
-          typename: lConf.typeName,
           outputFormat,
           srsname: lConf.projection
```

Comment:
```suggestion
          srsName: lConf.projection
```

Same remark as below